scripts/dev/dev-rebuild.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+PAYARA_DIR=/usr/local/payara5
+ASADMIN=$PAYARA_DIR/glassfish/bin/asadmin
+DB_NAME=dvndb
+DB_USER=dvnapp
+
+echo "Checking if there is a war file to undeploy..."
+LIST_APP=$($ASADMIN list-applications -t)
+OLD_WAR=$(echo $LIST_APP | awk '{print $1}')
+NEW_WAR=target/dataverse*.war
+if [ ! -z $OLD_WAR ]; then
+  $ASADMIN undeploy $OLD_WAR
+fi
+
+echo "Stopping app server..."
+$ASADMIN stop-domain
+
+echo "Deleting \"generated\" directory..."
+rm -rf $PAYARA_DIR/glassfish/domains/domain1/generated
+
+echo "Deleting ALL DATA FILES uploaded to Dataverse..." 
+rm -rf $PAYARA_DIR/glassfish/domains/domain1/files
```

Comment:
since you can change this location, can this be determined from the jvm option instead of hardcoded?

Reply:
I had that thought, that perhaps some devs keep their data files somewhere other than default (domain1/files). I figured I'd wait until there's demand to make this configurable. We could either add a variable to the top called something like DATA_FILES_DIR or yes, perhaps pull it out from `asadmin list-jvm-options`.

Reply:
OK, I'm fine with pushing that to later. Could you a) add that as a todo comment and b) add a note in the docs to point out that files would have to be in the default location?

Reply:
Sure, fixed in ecde956db.